### ISSUE TITLE
https://vocabtst.nerc.ac.uk/?_profile=alt&_mediatype=application/rdf+xml

### ISSUE THREAD

**Issue by alko-k** [surroundaustralia/VocPrez-theme-nvs#129]
That is a great enhancement to the previous description @nicholascar 
Weirdly all the RDF generated has the server name inside instead of the host name: https://vocabtst.nerc.ac.uk/
Like here:
<rdf:Description rdf:about="https://livbodcvocabtst.bodc.me:8008/">
        <rdf:type rdf:resource="http://www.w3.org/ns/dcat#DataService"/>
        <dcterms:description xml:lang="en">A Content Negotiation by Profile-compliant service that provides access to all of this catalogue's information</dcterms:description>
        <dcterms:title>System ConnegP API</dcterms:title>
        <dcterms:type rdf:resource="http://purl.org/dc/dcmitype/Service"/>
        <dcat:endpointURL rdf:resource="https://livbodcvocabtst.bodc.me:8008/"/>

**Comment by nicholascar** [comment]
That’s correct because the vocab.nerc.ac.uk identifies the dataset where as the vocabdev  URIs identify the DataService (distributions).

The identifier of the conceptual dataset never changes, even when the implementation location, e.g. dev/prod does.
